crates/services/p2p/src/gossipsub/topics.rs

```diff
@@ -10,22 +10,26 @@ use super::messages::{
 };
 
 pub const NEW_TX_GOSSIP_TOPIC: &str = "new_tx";
+pub const TX_PRECONFIRMATIONS_GOSSIP_TOPIC: &str = "tx_confirmations";
```

Comment:
question: why it's called both "confirmations" and "preconfirmations"?

Reply:
Holdover from earlier naming. Good catch.